themes/vpge/lib/scss/components/grad-grow-filter/_grad-grow-filter.scss

```diff
@@ -182,6 +197,65 @@
         }
       }
     }
+
+    .pager {
+      @include responsive-spacing('padding-top', 5);
+
+      .pager__items {
+        margin-bottom: 0;
+        padding-left: 0;
+
+        .pager__item {
+          margin: 1.5rem 1.5rem 0;
+          padding: 0.5rem 0.5rem 0;
+
+          @include grid-media-max('sm') {
+            margin: 1rem 0;
+            padding: 1.1rem 1.1rem 0;
+          }
+
+          &.pager__item--active {
+            border-bottom: 2px solid $su-color-black;
+
+            a {
+              color: $su-color-black;
+            }
+          }
+
+          &.pager__item--ellipsis {
+            position: relative;
+            bottom: 6px;
+            
+            &:hover,
+            &:focus,
+            &:active {
+              border-bottom: none;
+            }
```

Comment:
The ellipse is not focusable. by moving the other hover, focus classes to the proper element, this stuff can go away.